src/IconWithLabel/IconWithLabel.tsx

```diff
@@ -8,11 +16,13 @@ import { IconWithLabelProps } from "./IconWithLabel.types";
  * @param icon The icon to be displayed
  * @param label The label to render alongside the icon
  * @param href When defined the label is a clickable link to this url
+ * @param tooltip When defined it shows tooltip over the icon
  */
 export default function IconWithLabel({
   icon,
   label,
-  href
+  href,
+  tooltip = ""
```

Comment:
I think it is better to be more explicit here

```suggestion
  tooltip = undefined
```